programs/native/bpf_loader/src/lib.rs

```diff
@@ -29,6 +29,53 @@ fn dump_program(key: &Pubkey, prog: &[u8]) {
     }
 }
 
+#[allow(unused_variables)]
```

Comment:
Put underscores in front of the unused parameters instead?

Reply:
I'm not partial, just following an example that Vines set, underlines actually make for more compact code, maybe change later.

Reply:
ok

Reply:
the underscores would ensure you're notified when one of the remaining parameters is accidentally unused (typically as a result of shadowing)

Reply:
agreed, ok I'll change as part of this pr